.gitignore

```diff
@@ -5,6 +5,7 @@
 build/
 bpk-fonts-tmp/
 Example/Backpack/Resources/SkyscannerRelative*
+Fonts/Assets/SkyscannerRelative*
```

Comment:
Ignoring the new path that fonts are stored in. I'll leave the old path for now, incase anyone still has them there